packages/libproj/meta.yaml

```
@@ -0,0 +1,26 @@
+package:
+  name: libproj
+  version: 8.2.1
+
+source:
+  sha256: 76ed3d0c3a348a6693dfae535e5658bbfd47f71cb7ff7eb96d9f12f7e068b1cf
+  url: https://download.osgeo.org/proj/proj-8.2.1.tar.gz
+
+build:
+  library: true
+  script: |
+    mkdir -p build;
+    cd build \
+        && LDFLAGS="-s NODERAWFS=1 -sUSE_ZLIB=1 -sFORCE_FILESYSTEM=1" emcmake cmake ../ \
+        -DCMAKE_INSTALL_PREFIX=../../proj \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_CURL=OFF \
+        -DENABLE_TIFF=OFF \
+        -DBUILD_APPS=OFF \
+        -DBUILD_SHARED_LIBS=OFF \
+        -DBUILD_TESTING=OFF \
+        -DSQLITE3_LIBRARY=$PYODIDE_ROOT/cpython/build/sqlite-autoconf-3380500/sqlite3.wasm \
```

I think this should point to a `.a` file.

```suggestion
        -DSQLITE3_LIBRARY=$PYODIDE_ROOT/cpython/build/sqlite-autoconf-3380500/.libs/libsqlite3.a \
```

---

Ah, I didn't think to check `.libs`.